operator/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+- calvin0327
+- Poor12
+- lonelyCZ
+approvers:
+- RainbowMango
```

Comment:
```suggestion
reviewers:
- calvin0327
- lonelyCZ
- Poor12
approvers:
- calvin0327
- lonelyCZ
- Poor12
```

I think you should be the owner of the operator, thanks a lot for all your hard work!

Reply:
@RainbowMango thanks. 😄